spec/models/course/assessment/submission_spec.rb

```diff
@@ -77,6 +90,19 @@
       end
     end
 
+    describe '.from_course' do
+      let(:new_course) { create(:course) }
+      let(:new_assessment) { create(:course_assessment_assessment, course: new_course) }
+      let!(:new_submission) { create(:course_assessment_submission, assessment: new_assessment) }
```

Comment:
Is it better to avoid the `!` by calling `new_submission` in the `it` block after/before `submission1`?